### PR TITLE
Add glossary page on fingerprinting, link to it

### PR DESCRIPTION
**Pull request by wbamberg**
This adds a glossary page on fingerprinting and links to it from almost everywhere we use the term.

I didn't update the WebExtensions docs, although they do use the term, because AFAIK WebExt does not want Prettier formatting, so it would be a bunch more work to special-case those files.

### CODE REVIEW

File: files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The [Gamepad API](/en-US/docs/Web/API/Gamepad_API) introduces new events on the
 
 When a new gamepad is connected to the computer, the focused page first receives a {{ domxref("Window/gamepadconnected_event", "gamepadconnected") }} event. If a gamepad is already connected when the page loaded, the {{ domxref("Window/gamepadconnected_event", "gamepadconnected") }} event is dispatched to the focused page when the user presses a button or moves an axis.
 
-> **Note:** In Firefox, gamepads are only exposed to a page when the user interacts with one with the page visible. This helps prevent gamepads being used for fingerprinting the user. Once one gamepad has been interacted with, other gamepads that are connected will automatically be visible.
+> **Note:** In Firefox, gamepads are only exposed to a page when the user interacts with one with the page visible. This helps prevent gamepads being used for [fingerprinting](/en-US/docs/Glossary/Fingerprinting) the user. Once one gamepad has been interacted with, other gamepads that are connected will automatically be visible.
```
Comment (dawei-wang) [inline files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md:18]:
```suggestion
> **Note:** In Firefox, gamepads are only exposed to a page when the user interacts with one with the page visible. This helps prevent gamepads from being used for [fingerprinting](/en-US/docs/Glossary/Fingerprinting) the user. Once one gamepad has been interacted with, other gamepads that are connected will automatically be visible.
```

File: files/en-us/glossary/fingerprinting/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- the browser version
+- the timezone and preferred language
+- the set of video or audio codecs that are available on the system
+- the fonts installed on the system
+- the state of the browser's settings
+- the computer's display size and resolution
+
+A website can retrieve information like this by executing JavaScript and CSS on the device, and by combining this data can often create a unique fingerprint for a browser, which can then be used to track users across the web.
+
+Web standards are designed in such a way as to minimise the ability of a website to collect identifying information, and browsers typically add their own protections as well.
```
Comment (dawei-wang) [inline files/en-us/glossary/fingerprinting/index.md:18]:
```suggestion
Web standards are designed in such a way as to minimize the ability of a website to collect identifying information, and browsers typically add their own protections as well.
```

**Review by dawei-wang — COMMENTED** [review]
typo and missing word

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**